### DISCUSSION TITLE
can u add cpu temperature like as gpu temperature on task manager?

### DISCUSSION THREAD

**Discussion by strongerr1234** [valinet/ExplorerPatcher#208] [General]
hi can u add cpu temperature to task manager like as gpu temperature on taskmanager

**Comment by valinet** [comment]
Hi

I don’t have experience in the area and currently no plan of implementing that. If you find something out, please consider maybe contributing it back to this project.

Thanks

**Comment by Gaurav-Original-ClassicShellTester** [comment]
@strongerr1234 The best app for this is TechPowerUp ThrottleStop. It can put both CPU and GPU temperature in the tray directly.

There's also CoreTemp (for CPU temperature, power consumption and clock speed in the tray) or Open Hardware Monitor (for SSD/HDD temperature in the tray).
